pkg/controller/direct/certificatemanager/dnsauthorization_controller.go

```diff
@@ -107,9 +95,10 @@ func (m *model) AdapterForObject(ctx context.Context, reader client.Reader, u *u
 	}
 
 	// Get location
-	// TODO: Add location when field is added
-	// location := obj.Spec.Location
-	location := "global"
+	location := obj.Spec.Location
+	if location == "" {
+		location = "global"
+	}
 
 	var id *CertificateManagerDNSAuthorizationIdentity
 	// TODO: Add ExternalRef when field is added
```

Comment:
are the logs on 104 still relevant ?

Reply:
ack, I can change it later. Thank you for the review